libyul/optimiser/README.md

```diff
@@ -1,651 +1,5 @@
-Note that the Yul optimiser is still in research phase. Because of that,
-the following description might not fully reflect the current or even
-planned state of the optimiser.
+# Yul-Based Optimizer
```

Comment:
Nit: The filename in this diff still retains old spelling.

Reply:
yes the whole folder is called "optimiser" - should we rename it to optimizer?

Reply:
I think it's very low priority to rename internal folders and file names.

Reply:
Ok!